build.ps1

```diff
@@ -268,6 +269,7 @@ function cleanExtension([string] $bitness) {
     $keepRuntimes = @('win', 'win-x86', 'win10-x86', 'win-x64', 'win10-x64')
     Get-ChildItem "$privateSiteExtensionPath\$bitness\workers\powershell\runtimes" -Exclude $keepRuntimes -ErrorAction SilentlyContinue |
         Remove-Item -Recurse -Force -ErrorAction SilentlyContinue
+    deleteDuplicateWorkers
```

Comment:
Undo this. Move it back to line 217. This needs to be called only once after both 32bit and 64bit folders are cleaned